app/shared/scripts/HubSpotScript.js

```diff
@@ -0,0 +1,14 @@
+import Script from 'next/script'
+
+const HubSpotScript = () => {
```

Comment:
can we give the file and the component a more descriptive name? we can have multiple hubspot scripts